featurebyte/query_graph/node/generic.py

```diff
@@ -924,6 +930,12 @@ def _derive_sdk_code(
         return statements, out_var_name
 
 
+class LookupTargetNode(BaseLookupNode):
```

Comment:
Should this have a field for the point in time offset parameter or is the plan to add that later?

Reply:
hm, yea i can add that now!